src/UbiHttp.cpp

```diff
@@ -319,3 +319,8 @@ bool UbiHTTP::waitServerAnswer() {
  */
 
 void UbiHTTP::setDebug(bool debug) { _debug = debug; }
+
+tcpMap UbiHTTP::getMultipleValues(const char* deviceLabel, const char* variableLabels) {
```

Comment:
It is necessary to fit the new UbiProtocol sign. I have created these methods in the new abstract sign for the reasons below:

- The Particle compiler does not allow to have optional methods in the abstract sign, because of this, if a new method to retrieve data is needed the sign must be modified because of the actual library architecture.
- We will have, in the middle term, to implement this method in all the protocols. For now, it will be released just for TCP to fit a certain need of one of our customers.
- The protocol handler is in charge of avoiding to invoke this method by any other protocol rather than TCP